packages/now-build-utils/src/detect-builders.ts

```diff
@@ -971,11 +970,13 @@ function getRouteResult(
       }
 
       rewriteRoutes.push(...dynamicRoutes);
-      rewriteRoutes.push({
-        src: '^/api(/.*)?$',
-        status: 404,
-        continue: true,
-      });
+
+      if (ignoreRuntimes === undefined) {
```

Comment:
Probably just me being pedantic:

```suggestion
      if (typeof ignoreRuntimes === 'undefined') {
```

Reply:
Done